aidial_adapter_openai/app.py

```diff
@@ -75,7 +74,12 @@ async def chat_completion(deployment_id: str, request: Request):
 
     if is_text_to_image_deployment(deployment_id):
         return await text_to_image_chat_completion(
-            data, upstream_endpoint, api_key, is_stream, file_storage
+            data,
+            upstream_endpoint,
+            api_key,
+            is_stream,
+            file_storage,
+            request.headers["authorization"],
```

Comment:
You need to handle the case when the header is unset.

Reply:
fixed